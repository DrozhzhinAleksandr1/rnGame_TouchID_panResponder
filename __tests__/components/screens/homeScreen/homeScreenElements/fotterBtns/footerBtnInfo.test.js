import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { FooterBtnInfo } from '../../../../../../app/components/screens/homeScreen/homeScreenElement/fotterbtns/footerBtnInfo';

it('create snapshot FooterBtnInfo', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<FooterBtnInfo isActive={true} setIsInfoOpened={() => { }} />);
    expect(toolbarSnapshot).toMatchSnapshot();
});