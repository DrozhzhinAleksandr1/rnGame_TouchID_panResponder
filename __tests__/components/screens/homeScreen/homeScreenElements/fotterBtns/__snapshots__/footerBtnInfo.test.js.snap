// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot FooterBtnInfo 1`] = `
<TouchableWithoutFeedback>
  <Component
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#1D2E3E",
          "borderWidth": 0.5,
          "height": 40,
          "justifyContent": "center",
          "marginHorizontal": 5,
          "width": 40,
        },
        Object {
          "borderColor": "#FFFFFF",
        },
      ]
    }
  >
    <InfoWhite />
  </Component>
</TouchableWithoutFeedback>
`;
